diff --git a/podcasts-data/src/schema.rs b/podcasts-data/src/schema.rs
index 03cbed0..88f1622 100644
--- a/podcasts-data/src/schema.rs
+++ b/podcasts-data/src/schema.rs
@@ -1,8 +1,11 @@
+#![allow(warnings)]
+
 table! {
     episodes (title, show_id) {
+        rowid -> Integer,
         title -> Text,
         uri -> Nullable<Text>,
         local_uri -> Nullable<Text>,
         description -> Nullable<Text>,
         epoch -> Integer,
         length -> Nullable<Integer>,
@@ -30,11 +33,7 @@ table! {
         uri -> Text,
         last_modified -> Nullable<Text>,
         http_etag -> Nullable<Text>,
     }
 }
 
-allow_tables_to_appear_in_same_query!(
-    episodes,
-    shows,
-    source,
-);
+allow_tables_to_appear_in_same_query!(episodes, shows, source);
